mantidproject/mantid
# Allow loading of logs after duplicate detected

**Description of work.**

Move the exception handling to inside the loops so that log
loading continues after e.g. a duplicate is detected. An issue
was spotted on OFFSPEC where a block called NSPECTRA had been
created and this conflicted with a sample log created by the
LoadRaw routine itself.

**To test:**

Load raw file for OFFSPEC run 42712 which will call LoadLogs but you will not see sample logs such as 
s1hc and s1vc which are visible in the corresponding nxs file. After merging PR you should see s1hc and s1vc in sample logs by both routes (raw and nxs) 

Fixes #22963 

Does this update require release notes?


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
